lib/dealership.rb

```diff
@@ -0,0 +1,48 @@
+require './lib/car'
+
+class Dealership
+  attr_reader :name, :addy, :inventory
+  def initialize(name, addy)
+    @name = name
+    @addy = addy
+    @inventory = []
+  end
+
+  def inventory_count
+    return @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if @inventory == []
+      return false
+    else
+      return true
+    end
+  end
+
+  def cars_by_make(type)
+    return @inventory.make(type)
```

Comment:
I think you were in the middle of this method; see if you can come back to this and finish it out! 
A hint - `#make` is not a method on `@inventory`; you will need to somehow iterate through the `@inventory` to find what you're looking for.